pkg/virt-controller/watch/vmi.go

```diff
@@ -251,22 +256,23 @@ func (i informalSyncError) RequiresRequeue() bool {
 }
 
 type VMIController struct {
-	templateService    services.TemplateService
-	clientset          kubecli.KubevirtClient
-	Queue              workqueue.RateLimitingInterface
-	vmiInformer        cache.SharedIndexInformer
-	vmInformer         cache.SharedIndexInformer
-	podInformer        cache.SharedIndexInformer
-	pvcInformer        cache.SharedIndexInformer
-	topologyHinter     topology.Hinter
-	recorder           record.EventRecorder
-	podExpectations    *controller.UIDTrackingControllerExpectations
-	vmiExpectations    *controller.UIDTrackingControllerExpectations
-	dataVolumeInformer cache.SharedIndexInformer
-	cdiInformer        cache.SharedIndexInformer
-	cdiConfigInformer  cache.SharedIndexInformer
-	clusterConfig      *virtconfig.ClusterConfig
-	cidsMap            *cidsMap
+	templateService      services.TemplateService
+	clientset            kubecli.KubevirtClient
+	Queue                workqueue.RateLimitingInterface
+	vmiInformer          cache.SharedIndexInformer
+	vmInformer           cache.SharedIndexInformer
+	podInformer          cache.SharedIndexInformer
+	pvcInformer          cache.SharedIndexInformer
+	storageClassInformer cache.SharedIndexInformer
```

Comment:
Have to add this to `cache.WaitForCacheSync` call

Reply:
@jean-edouard maybe you missed this?

Reply:
Isn't that what you meant?
https://github.com/kubevirt/kubevirt/pull/11373/files#diff-3b1b828044b00dce368b3baa4e8947bef920133d86f717b788d4e9504a7c5749R27

Reply:
Here as well:  https://github.com/jean-edouard/kubevirt/blob/bindbackendstorage/pkg/virt-controller/watch/vmi.go#L284-L292

Reply:
hey @jean-edouard would love to help you get this in.  see above comment

Reply:
:facepalm: sorry for the delay, will address today, thanks for the ping!

Reply:
Pushed, PTAL!